pipelines/resources/deploy-ztp-spokes.yaml

```diff
@@ -131,6 +131,66 @@ spec:
       - name: ztp
         workspace: ztp
 
+    # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry-ocp
+    taskRef:
+      name: spoke-deploy-disconnected-registry-spokes-ocp
+    params:
+      - name: spokes-config
+        value: $(params.spokes-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+    runAfter:
+      - deploy-disconnected-registry
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+    # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry-olm
+    taskRef:
+      name: spoke-deploy-disconnected-registry-spokes-olm
+    params:
+      - name: spokes-config
+        value: $(params.spokes-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+    runAfter:
+      - deploy-disconnected-registry
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+    # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry-ps
```

Comment:
These `deploy-disconnected-registry-*` tasks are doing the actual mirror of the images, right? Can we rename them to reflect so? `mirror-olm-image` `mirror-ocp-images`? 

Maybe catalog is a better term?